pkg/util/requestbuffers.go

```diff
@@ -32,7 +34,7 @@ func NewRequestBuffers(p Pool) *RequestBuffers {
 
 // Get obtains a buffer from the pool. It will be returned back to the pool when CleanUp is called.
 func (rb *RequestBuffers) Get(size int) *bytes.Buffer {
-	if rb == nil {
+	if rb == nil || size > maxInPoolRequestBufferSize {
```

Comment:
Maybe this behaviour should also be mentioned in the doc comment.

Reply:
you mean CHANGELOG.md?

Reply:
No, I mean the documentation comment for the method, i.e. on line 35.